### PR TITLE
Upgrade to Substrate 0.9.25

### PR DESCRIPTION
**Pull request by vasylenko-yevhen**
- update standalone chain (node, runtime)
- update parachain (node, runtime)
- update primitives
- update modules (asset-registry)
- update gamedao protocol

### CODE REVIEW

File: bin/zero/cli/src/chain_spec.rs [hunk 1]
```diff
@@ -66,16 +66,11 @@ pub struct Extensions {
 
 /// Specialized `ChainSpec`.
 pub type ChainSpec = sc_service::GenericChainSpec<GenesisConfig, Extensions>;
-/// SUBZERO testnet generator
-pub fn subzero_config() -> Result<ChainSpec, String> {
-	ChainSpec::from_json_bytes(&include_bytes!("../../../res/alphaville.json")[..])
+/// Flaming Fir testnet generator
```
Comment (vasylenko-yevhen) [inline bin/zero/cli/src/chain_spec.rs:69]: Drop Flaming Fir and add Alphaville here.

File: bin/zero/cli/src/command.rs [hunk 2]
```diff
@@ -16,15 +16,24 @@
...
+use zero_runtime::RuntimeApi;
 use sc_cli::{ChainSpec, Result, RuntimeVersion, SubstrateCli};
 use sc_service::PartialComponents;
 
+use std::sync::Arc;
+
 impl SubstrateCli for Cli {
 	fn impl_name() -> String {
-		"S U B Z E R O".into()
+		"Substrate Node".into()
```
Comment (vasylenko-yevhen) [inline bin/zero/cli/src/command.rs:36]: Revert name

File: bin/zero/cli/src/command.rs [hunk 3]
```diff
@@ -49,15 +58,14 @@ impl SubstrateCli for Cli {
...
+			"" =>
+				return Err(
+					"Please specify which chain you want to run, e.g. --dev or --chain=local"
+						.into(),
+				),
 			"dev" => Box::new(chain_spec::development_config()),
 			"local" => Box::new(chain_spec::local_testnet_config()),
-			// default to alphaville when running without param to be compatible to old behaviour
-			"" | "subzero" | "alphaville" => Box::new(chain_spec::subzero_config()?),
+			"fir" | "flaming-fir" => Box::new(chain_spec::flaming_fir_config()?),
```
Comment (vasylenko-yevhen) [inline bin/zero/cli/src/command.rs:68]: Revert chains

File: bin/zero/executor/Cargo.toml [hunk 4]
```diff
@@ -1,44 +1,46 @@
 [package]
-name = "node-executor"
+name = "zero-executor"
 version = "3.0.0-dev"
-authors = ["ZERO <play@zero.io>"]
+authors = ["Parity Technologies <admin@parity.io>"]
```
Comment (vasylenko-yevhen) [inline bin/zero/executor/Cargo.toml:4]: Revert details for cargo files in zero node/runtime modules.

File: .docker/local-relay-parachain/docker-compose.yaml [hunk 5]
```diff
@@ -9,7 +9,7 @@ services:
       - "--node-key=0000000000000000000000000000000000000000000000000000000000000001"
       - "--validator"
```
Comment (vasylenko-yevhen) [inline .docker/local-relay-parachain/docker-compose.yaml:10]: Update version of the polkadot image.

**Review by vovacha — COMMENTED** [review]
It should have been split into parts, ex.: standalone chain, parachain, docker, modules (asset-registry). 100+ files is too much for the single PR.

**Review by vovacha — APPROVED** [review]
Good job! LGTM.

**Pull request merged**